oidc-controller/api/verificationConfigs/models.py

```diff
@@ -41,6 +41,7 @@ class VerificationProofRequest(BaseModel):
 class VerificationConfigBase(BaseModel):
     subject_identifier: str = Field()
     proof_request: VerificationProofRequest = Field()
+    include_v1_attributes: Optional[bool] = Field(default=False)
```

Comment:
This doesn't necessarily need to be optional. Thought it made sense this way as it is being removed eventually.

Reply:
I agree with the approach 👍🏻